Tocca.js

```diff
@@ -92,6 +93,7 @@
       data = data || {}
       data.x = currX
       data.y = currY
+      data.endingElement = document.elementFromPoint(currX, currY)
```

Comment:
the use of `document.elementFromPoint` to cache the `endingElement` is a hacky feature that I don't want to add to the library since it might lead to other bugs. You can achieve the same behaviour outside of this library by simply using the `event.data.x` `event.data.y` values

Reply:
> 
> 
> the use of `document.elementFromPoint` to cache the `endingElement` is a hacky feature that I don't want to add to the library since it might lead to other bugs. You can achieve the same behaviour outside of this library by simply using the `event.data.x` `event.data.y` values

I understand.  But what would be the alternative?  The person listening for the event would implement their own search based on the coordinates?  Because `document.elementFromPoint` was created for this reason.  

It could be useful is someone wants to track that a swipe went from `div` A to `div B` and then a specific action can be taken place.

Reply:
Furthermore, you can always check parents of the `endingElement` to find that a swipe ended up in the correct DOM tree you wanted.

Reply:
I think it's a safe feature to add.  OR we can add it as `estimatedEndingElement`

Reply:
>  But what would be the alternative? 

```js
const withEndingElement = fn => {
   return (event) => {
      event.endingElement = document.elementFromPoint(event.x, event.y)
      fn(event)
   } 
}

element.addEventListener('tap', withEndingElement(event => {
  // tada
  const { endingElement } = event
}))
```